assets/cubyz/sbb/mushroom/big/toadstool/base/0.zig.zon

```diff
@@ -0,0 +1,23 @@
+.{
+	.blueprint = "cubyz:mushroom/big/toadstool/base/0",
+	.children = .{
+		.white = .{
+			.{.structure = "cubyz:mushroom/big/toadstool/large_cap"},
+		},
+		.crimson = {
+			.{.structure = "cubyz:mushroom/big/toadstool/branch/red0", chance = 0.125},
+			.{.structure = "cubyz:mushroom/big/toadstool/branch/red1", chance = 0.125},
```

Comment:
Nothing to change here, just a note to make things easier for you in the future: These numbers don't need to add up to 1.